frontend/sw360-portlet/src/main/webapp/js/modules/codeScoop.js

```diff
@@ -0,0 +1,966 @@
+/*
+ * Copyright Siemens AG, 2017.
+ * Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+function Modal() {
+
+    var _this = this;
+
+    this.opened = false;
+    this.modalBox = undefined;
+    this.modalTitle = undefined;
+    this.modalContent = undefined;
+    this.modalOverlay = undefined;
+
+    this.init = function () {
+        var modalBox = document.createElement("div");
+        modalBox.style.display = "none";
+        modalBox.style.position = "fixed";
+        modalBox.style.left = "50%";
+        modalBox.style.top = "50%";
+        modalBox.style.transform = "translateX(-50%) translateY(-50%)";
+        modalBox.style.zIndex = "999999";
+        modalBox.style.border = "1px solid #c5c5c5";
+        modalBox.style.backgroundColor = "#ffffff";
+        modalBox.style.color = "#333333";
```

Comment:
Is this code generated? There are many magic strings, which could be moved to constants.

Reply:
@maxhbr basic logic that I do not want put them as a constants for now. this solution will still going in development for UI look&feel
as first iteration will have no any dependencies from sw360 css variables and will not be damaged by extra changes in sw360 css

Reply:
as I understood the issue, @maxhbr is pointing to this issue: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
@mcjaeger this is css hex color that sw360 have in codebase also. nothing magic here: just init css directly from js code without inheritance of sw360 basic styles to save my code from css conflicts

Reply:
(from discussion)
* just move the values to the const values to the beginning of that file.  (preferred)
* one solution is to create a separate JS object that contains all the style information (rather than having a css file (ok)
* separate css file which is either loaded all the times at page load or nastily loading a separate css at javascript runtime (not so good)